setup.cfg

```diff
@@ -41,6 +41,7 @@ testing =
     pytest
     pytest-rerunfailures
     pytest-xdist
+    tox
```

Comment:
This would install `tox` inside `tox`'s virtualenvs. Would it make more sense to update the contribution guide to suggest install tox before using it?